spec/support/organizations/breadcrumb_spec_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples_for 'validates presence of breadcrumbs' do
+  let(:html) { Nokogiri::XML(response.body, &:noblanks) }
+
+  it 'validates that all breadcrumbs are present' do
+    last_breadcrumb_index = breadcrumbs.length
+    last_breadcrumb = breadcrumbs.pop
+
+    breadcrumbs.each_with_index do |breadcrumb, index|
```

Comment:
Why not call it directly `expected_breadcrums` so you don't have to add the `breadcrumbs` let in the spec itself?